de.xwic.appkit.webbase/src/de/xwic/appkit/webbase/editors/mappers/NumericInputboxMapper.java

```diff
@@ -16,27 +16,28 @@
  *******************************************************************************/
 package de.xwic.appkit.webbase.editors.mappers;
 
-import de.jwic.controls.RadioGroup;
+import de.jwic.controls.InputBox;
+import de.jwic.controls.NumericInputBox;
 import de.xwic.appkit.core.config.model.EntityDescriptor;
 import de.xwic.appkit.core.config.model.Property;
 import de.xwic.appkit.core.dao.IEntity;
+import de.xwic.appkit.core.dao.ValidationResult.Severity;
 import de.xwic.appkit.core.model.queries.IPropertyQuery;
 import de.xwic.appkit.webbase.editors.ValidationException;
-import de.xwic.appkit.webbase.editors.builders.EYesNoRadioBuilder;
 
 /**
- * Mapper for the InputBox control.
+ * Mapper for the NumericInputBox control.
  *
  * @author Aron Cotrau
```

Comment:
Again - Change the author to yourself - or delete the tag if you want to remain anonymous